compressors/prefix_free_compressors.py

```diff
@@ -122,7 +122,8 @@ def __init__(self, prob_dist: ProbabilityDist):
         """
         create the prefix free tree
         """
-        self.root_node = self.build_tree(prob_dist)
+        self.prob_dist = prob_dist
+        self.root_node = PrefixFreeTreeNode(id=None, code=None)
 
     def get_encoding_table(self):
```

Comment:
I had added this code for encoding_table.. but as you mentioned yday, it is un-intuitive to have this a function of the `PrefixFreeNode`. I will check later if this can be fixed and moved here, as that is more that.

Reply:
https://github.com/kedartatwawadi/stanford_compression_library/issues/38 added an issue for this